### PR TITLE
{vis}[GCCcore/11.3.0] Pango v1.50.7, FriBidi v1.0.12, ICU v71.1, ...

### PR DESCRIPTION
**Pull request by jfgrimm**
requires:

- [x] #15439 
- [x] #15437

(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node031.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/38ce3f8686ee888ca31e8bdefb8da87d for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 0 out of 0 (4 easyconfigs in total)
login2.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/ffa1a632f1dec270174138d88f81ec52 for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/f/FriBidi/FriBidi-1.0.12-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
```
Comment (verdurin) [inline easybuild/easyconfigs/f/FriBidi/FriBidi-1.0.12-GCCcore-11.3.0.eb:1]: Please restore the previous author info.

File: easybuild/easyconfigs/h/HarfBuzz/HarfBuzz-4.2.1-GCCcore-11.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
```
Comment (verdurin) [inline easybuild/easyconfigs/h/HarfBuzz/HarfBuzz-4.2.1-GCCcore-11.3.0.eb:1]: Same here with the author info, though fine to note that it was for previous versions.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/3e7f0c19d0828857a60f52eb3404b8f8 for a full test report.

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/f692d94dbba89a0694b1df2fcbbd1894 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/6e8bfd23f5bf3588870e1617d28d58e0 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
bear-pg0306u11a.bear.cluster - Linux RHEL 8.5, POWER, 8335-GTX (power9le), 4 x NVIDIA Tesla V100-SXM2-16GB, 470.57.02, Python 3.6.8
See https://gist.github.com/feb4e2602468f52a2e373ef5f0d08b9c for a full test report.

File: easybuild/easyconfigs/h/HarfBuzz/HarfBuzz-4.2.1-GCCcore-11.3.0.eb (hunk 2 above)
Comment (jfgrimm) [reply easybuild/easyconfigs/h/HarfBuzz/HarfBuzz-4.2.1-GCCcore-11.3.0.eb:1]: the credit was present for one version, and that isn't the most recent?

Comment (verdurin) [reply easybuild/easyconfigs/h/HarfBuzz/HarfBuzz-4.2.1-GCCcore-11.3.0.eb:1]: Yes, it isn't always super-clear in the `review` output, and as above it looks like it was already removed.

I prefer to check this, as I know for some people it's a condition of their contributions that they add the authorship.

File: easybuild/easyconfigs/f/FriBidi/FriBidi-1.0.12-GCCcore-11.3.0.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/f/FriBidi/FriBidi-1.0.12-GCCcore-11.3.0.eb:1]: Yes, seems it was already removed.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @jfgrimm!

**Pull request merged**